api/src/wfl/module/covid.clj

```diff
@@ -346,11 +351,21 @@
                       ["id = ?" id])))
     (throw (ex-info "No successful workflows in queue" {:executor executor}))))
 
+(defn ^:private terra-executor-workflows
+  [{:keys [workspace details] :as _executor}]
+  (->> (jdbc/with-db-transaction [tx (postgres/wfl-db-config)]
+         (when-not (postgres/table-exists? tx details)
+           (throw (ex-info "Missing executor details table" {:table details})))
+         (->> (format "SELECT DISTINCT rawls_submission_id FROM %s" details)
+              (jdbc/query tx)))
+       (mapcat (comp :workflows #(firecloud/get-submission workspace %)))))
+
 (defoverload create-executor! terra-executor-name create-terra-executor)
 (defoverload update-executor! terra-executor-type update-terra-executor)
 (defoverload load-executor!   terra-executor-type load-terra-executor)
 (defoverload peek-queue!      terra-executor-type peek-terra-executor-queue)
 (defoverload pop-queue!       terra-executor-type pop-terra-executor-queue)
+(defoverload executor-workflows terra-executor-type terra-executor-workflows)
```

Comment:
Awesome, I look forward to using this for [GH-1315](https://broadinstitute.atlassian.net/browse/GH-1315).